### PR TITLE
feat(docs): add full examples for spark tasks

### PR DESCRIPTION
**Pull request by shrutimantri**

### CODE REVIEW

File: src/main/java/io/kestra/plugin/spark/JarSubmit.java [hunk 1]
```diff
@@ -25,12 +25,19 @@
...
-                "runner: DOCKER",
-                "master: spark://localhost:7077",
-                "mainResource: {{ inputs.file }}",
-                "mainClass: spark.samples.App",
-            }
+            full = true,
+            code = """
+                id: spark_jar_submit
+                namespace: company.name
+
```
Comment (wrussell1999) [inline src/main/java/io/kestra/plugin/spark/JarSubmit.java:32]: Add an input of type `FILE`

File: src/main/java/io/kestra/plugin/spark/PythonSubmit.java [hunk 2]
```diff
@@ -31,40 +31,47 @@
...
-            }
+            full = true,
+            code = """
+                id: spark_python_submit
+                namespace: company.name
+
+                tasks:
+                  - id: python_submit
+                    type: io.kestra.plugin.spark.PythonSubmit
+                    runner: DOCKER
```
Comment (wrussell1999) [inline src/main/java/io/kestra/plugin/spark/PythonSubmit.java:42]: Does `taskRunner` work for this task? If so, we should replace `runner` with it.

Same for `docker` with `taskRunner` and `containerImage`

File: src/main/java/io/kestra/plugin/spark/RSubmit.java [hunk 3]
```diff
@@ -28,20 +28,27 @@
...
-            }
+            full = true,
+            code = """
+                id: spark_r_submit
+                namespace: company.name
+
+                tasks:
+                  - id: r_submit
+                    type: io.kestra.plugin.spark.RSubmit
+                    runner: DOCKER
```
Comment (wrussell1999) [inline src/main/java/io/kestra/plugin/spark/RSubmit.java:39]: Replace with `taskRunner` if it works

File: src/main/java/io/kestra/plugin/spark/PythonSubmit.java (hunk 2 above)
Comment (shrutimantri) [reply src/main/java/io/kestra/plugin/spark/PythonSubmit.java:42]: Let us take this as a separate issue / PR as I will have to install Spark to check this out. This looks to be a bit different as I do not see image mentioned here.

File: src/main/java/io/kestra/plugin/spark/RSubmit.java (hunk 3 above)
Comment (shrutimantri) [reply src/main/java/io/kestra/plugin/spark/RSubmit.java:39]: Let us take this as a separate issue / PR.

File: src/main/java/io/kestra/plugin/spark/PythonSubmit.java (hunk 2 above)
Comment (wrussell1999) [reply src/main/java/io/kestra/plugin/spark/PythonSubmit.java:42]: Issue made: https://github.com/kestra-io/plugin-spark/issues/72

**Review by wrussell1999 — APPROVED** [review]

**Pull request merged**